packages/plugin-core-example/package.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
With our building system, this example plugin would also get published. This is probably not what we want. We could replace this with a useful simple plugin.

Additionally, the paths used in this plugin all refer to local references. This might be confusing for people copying the template to create their own. We could develop a command in our CLI to generate this structure for them or make a tutorial on the website where we guide them on how to build a simple plugin. The core plugins would then just serve as examples.

Reply:
I can make this one private.
If you have a suggestion for a simple plugin that would be better indeed.
I like the idea of a template generator, I made an issue for this (#166)

Reply:
Removed it.